app/code/Magento/InventoryCache/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "magento/module-cache",
```

Comment:
I guess we have some naming issue here. Should it be `magento/module-inventory-cache` or so?